sdk/openai/Azure.AI.OpenAI/src/Custom/OpenAIClient.cs

```diff
@@ -17,40 +17,256 @@ namespace Azure.AI.OpenAI
     /// <summary> Azure OpenAI APIs for completions and search. </summary>
     public partial class OpenAIClient
     {
+        private const string publicOpenAIVersion = "1";
+        private const string publicOpenAIEndpoint = $"https://api.openai.com/v{publicOpenAIVersion}";
+
+        /// <remarks> This is the deployment id for all Azure OpenAI operations (completions and embeddings) </remarks>
+        private string DeploymentId { get; } = "";
```

Comment:
In .NET, when a string does not have a value, it is generally preferred to return `null` rather than an empty string.